content/en/docs/reference/node/cri-api-development-policies.md

```diff
@@ -0,0 +1,107 @@
+---
+content_type: "reference"
+title: Kubernetes feature development and Container runtimes
+weight: 10
+---
+
+The mechanics of a feature development that requires new CRI APIs is covered in
+documentation on CRI API [feature-development](https://github.com/kubernetes/cri-api?tab=readme-ov-file#feature-development).
```

Comment:
In end user docs we avoid hyperlinking to pages on GitHub, especially anything in-project.